products/ocp4/profiles/pci-dss-4-0.profile

```diff
@@ -18,8 +18,6 @@ description: |-
 
 filter_rules: '"ocp4-node" not in platforms and "ocp4-master-node" not in platforms and "ocp4-node-on-sdn" not in platforms and "ocp4-node-on-ovn" not in platforms'
 
-# Req-2.2
-extends: cis
```

Comment:
Is it safe to assume this is the old way of doing things, and what you did on https://github.com/ComplianceAsCode/content/pull/12309/commits/2438f1235d4626995b7909381f047e6f83e943af#diff-67844ce694c84c54b76dac7610bc6443a329477c4a685417d3aaa1eae7b6e29cR391 is how we should be doing it moving forward?

Reply:
I guess so, if one wants to take advantage of auto-referencing in control files.